From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 5 Feb 2019 09:32:57 -0500
Subject: Update css node tests

The GtkSwitch now has two image nodes.

(cherry picked from commit df77965153c1f1940407c167e92b56b19e0a3769)
---
 testsuite/css/nodes/buttons.nodes | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/testsuite/css/nodes/buttons.nodes b/testsuite/css/nodes/buttons.nodes
index 418a132..e19da6a 100644
--- a/testsuite/css/nodes/buttons.nodes
+++ b/testsuite/css/nodes/buttons.nodes
@@ -13,9 +13,13 @@
       radio:dir(ltr)
       label:dir(ltr)
     switch:dir(ltr):checked
-      slider
+      slider:dir(ltr):checked
+      image:dir(ltr):checked
+      image:dir(ltr):checked
     switch:dir(ltr)
       slider
+      image:dir(ltr)
+      image:dir(ltr)
     button.text-button:dir(ltr)
       label:dir(ltr)
     button.image-button:dir(ltr)
